app/controllers/curiosity_cards_controller.rb

```diff
@@ -4,7 +4,7 @@ class CuriosityCardsController < ApplicationController
   before_action :find_curiosity_scope_by_current_user, only: [:update, :destroy]
 
   def index
-    @curiosities = CuriosityCard.all
+    @curiosities = CuriosityCard.order_curiosity_by_desc.page(params[:page]).per(12)
```

Comment:
Model name is `CuriosityCard`. That being said, `order_curiosity_by_desc` sound redundant and not so clear 
Maybe just `ordered_by_recents`? 🤔